keyboards/cannonkeys/hoodrowg/rules.mk

```
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = lite	# Virtual DIP switch configuration
```

Is `lite` still supported? I'm attempting to `make` this locally and it's telling me its not valid.

---

Nope.  only "yes" or "no" is supported now. And "yes" is lite.

---

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```